keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -193,31 +218,65 @@ for new profile sources to be added in the future (e.g. Kubernetes predefined pr
 profiles). The seccomp options struct leaves room for future extensions, such as defining the
 behavior when a profile cannot be set.
 
-<<[UNRESOLVED]>>
-What to do with the localhost profile type, given that we want to deprecate it? Alternative for
-consideration:
+##### LocalhostProfile
+
+This KEP proposes we GA LocalhostProfile as the only source of user-defined profiles at this point. 
+User-defined profiles are essential for users to realize the full benefits out of seccomp, allowing 
+them to decrease their attack surface based on their own workloads. 
+
+###### Updating seccomp profiles
+Seccomp profiles are applied at container creation time. Therefore, updating the file contents of a 
+profile on disk after that point will not cause any changes to the running containers that are using 
+it, but will apply the updated profile on container restart. Note that amending such files is not 
+recommended and may cause containers to fail on next restart, in the case of the new profile being more 
+restrictive, invalid or the file no longer being present on disk.
+
+Currently, users have no way to tell whether their physical profiles have been deleted or modified.
+This KEP proposes no changes to the existing functionality.
+
+The recommended approach for rolling out changes to seccomp profiles is to always create _new profiles_ 
+instead of updating existing ones. Create and deploy a new version of the existing Pod Template, changing the
+profile name to the newly created profile. Redeploy, once working delete the former Pod Template. This 
+will avoid disruption on in-flight workloads.
+
+The current behavior around updating profiles while they are being used, is not adequate for a built-in 
+security feature such as seccomp. However, given its current "pseudo-GA" state, we don't want to change it 
+with this KEP. We will explore improvements to the behavior through the seccomp-operator and/or a new 
+feature-gated improvement.
```

Comment:
I will rephrase it as it is not exactly what I meant. For me the existing issues are:

- LocalhostProfile files are not sync'ed across nodes.
- The operation/maintenance of localhost profiles is not ideal, as a cluster admin may not have an easy way to identify what pods have been restarted since the last profile change. 

We are aiming to initially solve both problems out of tree with the seccomp-operator.